src/features/dashboard/components/SyncUpdateButton/SyncUpdateButton.tsx

```
@@ -0,0 +1,28 @@
+const SyncUpdateButton = () => {
+  const { data } = useProfileQuery();
+  const router = useInternalRouter();
+  const formattedLastSyncedAt = data.lastUpdatedAt ? format(new Date(data.lastUpdatedAt), 'yy년 M월 d일 HH:mm') : '';
+
+  const handleResyncLogin = () => {
+    router.push(ROUTES.RESYNC.LOGIN);
+  };
+
+  return (
+    <button className={clsx(styles.container, styles.text, '--body-sm-medium')} onClick={handleResyncLogin}>
+      {formattedLastSyncedAt} 업데이트
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)